src/scenes/PlantsDashboardRouter/PlantsDashboardView.tsx

```diff
@@ -95,6 +143,14 @@ export default function PlantsDashboardView(): JSX.Element {
     }
   }, [dispatch, org.selectedOrganization.id, selectedPlantingSiteId]);
 
+  useEffect(() => {
+    if (
+      lastObservationHistory?.data?.boundary?.coordinates !== previousObservationHistory?.data?.boundary?.coordinates
+    ) {
+      // verify if we need to do this validation
```

Comment:
Temporary comment

Reply:
Yes, I still have to add the comparison here that's why the comment is there